src/components/SiteInfoInputForm.js

```diff
@@ -0,0 +1,786 @@
+import React, {Component} from 'react';
+import {Alert, Picker, Platform, ScrollView, View, Image} from 'react-native';
```

Comment:
This is now the reusable component that both `AddSiteFormScreen` and `EditSiteScreen` use.